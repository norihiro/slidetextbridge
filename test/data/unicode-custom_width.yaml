steps:
  - type: stdin

  - type: linebreak
    # ambiguous_char_width: 1 # default
    custom_width:
      'あ': 3
      'か': 7
    split_long_line: 10

  - type: stdout
