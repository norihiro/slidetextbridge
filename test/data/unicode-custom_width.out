あいうえ
お

かき
くけこ

さしすせそ

